bundle/manifests/dscinitialization.opendatahub.io_dscinitializations.yaml

```diff
@@ -46,7 +46,7 @@ spec:
             description: DSCInitializationSpec defines the desired state of DSCInitialization
             properties:
               applicationsNamespace:
-                default: opendatahub
+                default: odh
```

Comment:
Should this be `opendatahub-system`

Reply:
you are right! not sure why it generated to "odh" but not "opendatahub-system"